docs/websocket-protocol.rst

```diff
@@ -0,0 +1,45 @@
+websocket protocol
+====================
+
```

Comment:
Maybe for another time, but I think more information needs to be added in here to describe the different parameters

Reply:
Agreed. Writing a ticket.